src/ntfs.rs

```diff
@@ -105,6 +105,8 @@ struct StopTime {
     #[serde(default, deserialize_with = "de_from_u8", serialize_with = "ser_from_bool")]
     datetime_estimated: bool,
     local_zone_id: Option<u16>,
+    headsign: Option<String>,
+    stop_time_id: Option<String>,
```

Comment:
StopTime is the object that use the RAM. Thus, what's the impact of this change on fr-idf?

We may need to manage these data from outside (with something like a `stop_time_ids: HashMap<(Idx<VehicleJourney>, u32), String>`) as these fields are rarely used.

Reply:
without stop_time_id and headsign
1.03 GB

without stop_time_id
1.40 GB

with stop_time_id and headsign
2.17 GB :open_mouth:

I remove these 2 lines, I will do something better as you suggest in another PR

Reply:
That's almost always None, thus an external storage will be less costy

Reply:
It's not very used for the moment, but I think it will be more and more used in the future.